database/things/kotlin.pldb

```diff
@@ -39,6 +39,7 @@ features
  hasMacros false
  hasOperatorOverloading true
  hasSemanticIndentation false
+ hasCaseInsensitiveIdentifiers false
```

Comment:
Is this correct? I could not find anything on https://en.wikipedia.org/wiki/Kotlin_(programming_language), but it appears that it is case-insensitive.

Reply:
https://rosettacode.org/wiki/Case-sensitivity_of_identifiers#Kotlin says that it is case-sensitive.

Reply:
Suggest about it too @celtic-coder